src/main/java/ai/lucidtech/las/sdk/NameAndDescriptionOptions.java

```diff
@@ -0,0 +1,43 @@
+package ai.lucidtech.las.sdk;
+
+import org.json.JSONObject;
+
+
+public class NameAndDescriptionOptions<T> {
+    private String name;
+    private String description;
+    private static final String NOT_PROVIDED = "NOT PROVIDED";
+
+
+    public NameAndDescriptionOptions(){
+        this.name = NOT_PROVIDED;
+        this.description = NOT_PROVIDED;
+    }
+
+    public T setName(String name){
+        assert name != this.NOT_PROVIDED : "name must have another value";
```

Comment:
I think Java string compare is done with .equals(). However I think using != achieves desired semantics here